tileserver/__init__.py

```diff
@@ -38,12 +40,26 @@ def coord_is_valid(coord):
     return True
 
 
-RequestData = namedtuple('RequestData', 'layer_spec coord format')
+RequestData = namedtuple('RequestData', 'layer_spec coord format params')
```

Comment:
I'd say make it `tile_size` for now. If we find ourselves adding a bunch of stuff to it we can reorganize.